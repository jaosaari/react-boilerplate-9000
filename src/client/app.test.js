import App from './app'
import React from 'react'
import {shallow} from 'enzyme'

describe('App', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<App/>)

    expect(wrapper.find('div').text()).toBe('Welcome to the React Boilerplate 9000 application, state of the art in React Boilerplate applications!!!')
    expect(wrapper).toMatchSnapshot()
  })
})
