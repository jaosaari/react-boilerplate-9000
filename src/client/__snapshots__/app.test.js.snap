// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Welcome to the React Boilerplate 9000 application, state of the art in React Boilerplate applications!!!",
    },
    "ref": null,
    "rendered": "Welcome to the React Boilerplate 9000 application, state of the art in React Boilerplate applications!!!",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Welcome to the React Boilerplate 9000 application, state of the art in React Boilerplate applications!!!",
      },
      "ref": null,
      "rendered": "Welcome to the React Boilerplate 9000 application, state of the art in React Boilerplate applications!!!",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
